plasmapy/particles/particle_collections.py

```diff
@@ -199,8 +247,9 @@ def _get_particle_attribute(self, attr, unit=None, default=None):
 
     def append(self, particle: ParticleLike):
         """Append a particle to the end of the |ParticleList|."""
-        # TODO: use particle_input when it works with CustomParticle and ParticleLike
```

Comment:
Because `particle_input` can now create `ParticleList` objects, we want to avoid using `particle_input` here to avoid circularity.